src/tools/miri/tests/fail/stacked_borrows/zst_slice.rs

```diff
@@ -1,5 +1,5 @@
 //@compile-flags: -Zmiri-strict-provenance
-//@error-in-other-file: /retag .* tag does not exist in the borrow stack/
+//@error-in-other-file: unreachable
```

Comment:
You just made this test useless. It's a Stacked Borrows test, it must lead to a Stacked Borrows error. So this definitely cannot land.